roles/docker_storage_driver/tasks/main.yml

```diff
@@ -0,0 +1,19 @@
+---
```

Comment:
I'd move this task under your docker_storage_to_overlay2 role.  I don't think it warrants its own role.

Reply:
@kwoodson, @ashcrow: What about merging it into `openshift_facts`?  All it does is set a `docker_storage_driver` fact, but it may as well be `openshift.docker.storage_driver`.

[grow_docker_vg.yml](https://github.com/openshift/openshift-ansible/blob/master/playbooks/adhoc/grow_docker_vg/grow_docker_vg.yml) could also use it.

Reply:
We're stripping out the docker portions of openshift_facts.

I agree, this doesn't need to be it's own role.

Reply:
Does that mean `openshift.docker.service_name` is going away?  I just started using that at @ashcrow's suggestion.

Reply:
Yes, my intention is to remove everything openshift.docker and remove all openshift.x variables from the docker roles.

Reply:
@michaelgugino where will he be able to get the correct runtime service name from?

Reply:
@ashcrow we can either move away from a boolean-based set of container variables to a name based set of container variables, or we can just implement the var logic with set_fact or include_vars in the role(s) that need that information.  Either approach should be pretty simple, as the runtime service name is not that widely used.

Reply:
As long as we don't block @mbarnes work on stuff we are expecting to do in the future I think it's fair to move to another way of noting to containers and within the installer what the runtime service is named.